.travis.yml

```diff
@@ -42,6 +42,10 @@ jobs:
       if: type = push AND env(GITHUB_TOKEN) IS present
       script: test/license-test/run-license-test.sh
       name: License Test
+    - stage: Test
+      if: type = push AND env(GITHUB_TOKEN) IS present
+      script: test/e2e/run-test
```

Comment:
If you update this and license-test job to use `make` , then the whole file will be consistent 👍

Reply:
done!